data/profiles/remote.yaml

```diff
@@ -17,7 +17,10 @@ spec:
     enabled: false
 
   gateways:
-    enabled: false
+    enabled: true
+    components:
+      egressGateway:
+        enabled: false
```

Comment:
do you need to specify ingressgateway enabled true?

Reply:
no, because it will inherit the feature setting.